jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/explorer/CodeWhispererExplorerActionManager.kt

```diff
@@ -194,7 +202,8 @@ enum class CodeWhispererExploreStateType {
     HasShownHowToUseCodeWhisperer,
     DoNotShowAgainWarn,
     DoNotShowAgainError,
-    AccountlessNullified
+    AccountlessNullified,
+    ConnectionExpiredDoNotShowAgain
```

Comment:
none of these can ever be deleted right?

Reply:
For all the no-longer-used states if we want to delete them, we have to find a way to make it backward compatiable to reading the old on-disk states that have these fields.

Reply:
we have a couple of unit tests for making sure run configuration deserialization is backwards compatible. consider something similar in the future